x-pack/plugin/mapper-aggregate-metric/src/test/java/org/elasticsearch/xpack/aggregatemetric/mapper/AggregateDoubleMetricFieldMapperTests.java

```diff
@@ -499,79 +524,7 @@ public SyntheticSourceExample example(int maxVals) {
             return new SyntheticSourceExample(value, value, this::mapping);
         }
 
-        private Object randomAggregateMetric() {
-            if (malformedExample && randomBoolean()) {
-                return malformedValue();
-            }
-
-            return validMetrics();
-        }
-
-        private Object malformedValue() {
```

Comment:
These scenarios are moved to `exampleMalformedValues` which is not correctly used with synthetic source.